src/components/Sketches/index.js

```diff
@@ -82,6 +82,22 @@ class Sketches extends React.Component {
 
   renderSketches = () => {
     let newList = this.props.programs.concat([]);
+    if (newList.size === 0) {
+      return (
+        <div>
+          <div className="no-sketches-container">
+            <h2>There's nothing here! Why don't you try creating a sketch!</h2>
+            <p>
+              <SketchesButton
```

Comment:
Going to need to update this to the new button when you merge the style change

Reply:
technically second sentence is a question but I'm nitpicking

Reply:
> Going to need to update this to the new button when you merge the style change

Can't believe I didn't do it in this PR. It's a bit annoying since I think I'm going to get a merge conflict off of importing button, so I'll probably do this in whichever of the two branches I end up merging last.

Gonna tag #56 just so I remember.

Reply:
> technically second sentence is a question but I'm nitpicking

Fixed.